src/CarbonAware.Tools/CarbonAware.Tools.WattTimeClient/src/Configuration/WattTimeClientConfiguration.cs

```diff
@@ -23,6 +23,12 @@ public class WattTimeClientConfiguration
     /// </summary>
     public string BaseUrl { get; set; } = "https://api2.watttime.org/v2/";
 
+    /// <summary>
+    /// Gets or sets the cached expiration time (in seconds) for a BalancingAuthority instance.
+    /// It defaults to 300 secs.
+    /// </summary>
+    public int BACacheExpirationInSecs { get; set; } = 300;
```

Comment:
The WattTime docs recommend, `querying /ba-from-loc at least once a month with coordinates to determine a device's BA region.` 

We can probably use a more aggressive default value.  I would say at least an hour (3600), but I would be comfortable going as high as 1 day (86400).  What do you think?

Reply:
@bderusha , no preference tbh. I'll default it to 1 day then.